Cargo.toml

```diff
@@ -18,9 +18,11 @@ aurora-engine-sdk = { git = "https://github.com/aurora-is-near/aurora-engine.git
 aurora-engine-transactions = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
 aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
 base64 = "0.13.0"
+bitflags = "1.3"
 borsh = "0.9"
 ethabi = "17.2"
 hex = "0.4"
+near-plugins = { git = "https://github.com/aurora-is-near/near-plugins", rev = "d822c2f051ccdc03b50f206650524daf43ae7526" }
```

Comment:
I just create first tag for near-plugins: [v0.0.1](https://github.com/aurora-is-near/near-plugins/releases/tag/v0.0.1). Let's use it instead of the direct hash.

Reply:
Great, I'll use the tag and update this PR to be in line with the recent changes in the acl plugin.